scripts/create-snapcraft-yaml.jl

```diff
@@ -12,7 +12,7 @@ using JSON3
 release = ARGS[1]
 contents = JSON3.read(HTTP.get("https://julialang-s3.julialang.org/bin/versions.json").body)
 
-versions = VersionNumber.(String.(keys(contents)))
+versions = VersionNumber.(String.(keys(filter(p -> last(p)[:stable], contents))))
```

Comment:
You can simplify it a bit like this:
```suggestion
versions = [VersionNumber(String(k)) for (k, v) in contents if v.stable]
```